app/templates/components/event-card.hbs

```diff
@@ -23,10 +23,18 @@
       </SmartOverflow>
       <div class="meta">
         <span class="date">
-          {{general-date this.event.startsAt tz=this.event.timezone}}
+          {{general-date this.event.startsAt 'date-time-long'}}
+          -
+          {{general-date this.event.endsAt 'date-time-long'}}
         </span>
       </div>
       <SmartOverflow @class="description">
+        {{#if this.event.online}}
+          Online Event
+        {{/if}}
+        {{#if (and this.event.online this.event.shortLocationName)}}
```

Comment:
use this if statement inside upper if statement and then no need to add ```and``` just use like 
```handlebars
{{#if this.event.shortLocationName}}
       ...
       ...
{{/if}}
```

Reply:
if the event is not online then location will also not be displayed.

Reply:
thanks 😄